### ISSUE TITLE
Fig 10

### ISSUE THREAD

**Issue by markussmiettinen** [NMRLipids/NMRlipidsIVotherHGs#128]
Ivan Gushchin wrote per email: _two profiles for each simulation is not explained; no dashed lines despite caption "Number density profiles of Ca2+ (solid line) and Cl− (dashed line)"_

**Comment by markussmiettinen** [comment]
- I improved the dashing of the Cl- lines in #134. Should be clear now.
- I added the missing Cl- line to Lipid17/ff99 in #134.
- I checked, and the two profiles show the two leaflets, not two independent simulations. The caption could be changed to _Number density profiles of Ca2+ (solid line) and Cl− (dashed line) from POPC:POPS (5:1) mixtures simulated with different force fields. **Profiles for both leaflets shown.** The vertical green bars indicate the location of the phosphate density peak. The smallest simulated CaCl2 concentrations are shown. The **Ca2+** density profiles for all of the simulated concentrations are given in Figure S18._
